src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorClassLibrary-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Razor Class Library",
+    "id": "1023",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}"
+  },
+  "description": {
+    "text": "A project template for creating a Razor class library.",
+    "id": "1024",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}"
+  },
+  "order": 301,
+  "icon": "vs-2017.3/Empty.png",
```

Comment:
FYI: Temporarily using Empty project template's icon, but in discussions for a separate icon for this